common/bridge_info.json

```diff
@@ -1,12 +1,22 @@
 {
-    "child" : {
-        "url": "http://127.0.0.1:8551",
-        "key": "0xc544b44c1c58955af516c1f2ff17f8fd522604f1ea6b64db79e067343ed5e307",
-        "operator": "0x37ca5a315d5e17e428bb5edff2b6f2e6211bbe39"
+    "sender": {
+        "child": {
+            "key": "0xc544b44c1c58955af516c1f2ff17f8fd522604f1ea6b64db79e067343ed5e307"
+        },
+        "parent": {
+            "key": "0x4b07ca7412ad2bb0e62db30369b9f08a8724fb81fce4b3b1af23800233074fbf"
+        }
     },
-    "parent" : {
-        "url": "http://127.0.0.1:9781",
-        "key": "0x4b07ca7412ad2bb0e62db30369b9f08a8724fb81fce4b3b1af23800233074fbf",
-        "operator": "0xd5c0321bafc1d82ccd5cb60c0d943328c9d93318"
-    }
+    "bridges": [
+        {
+            "child" : {
+                "url": "http://127.0.0.1:8551",
```

Comment:
The `url` property is used to caver initialization. It seems reasonable to locate it in the sender's context. Please clarify why did you move it to `bridges`.

Reply:
@hyunsooda They are also used for `subbridge` RPCs. I thought the binding between one bridge's URL and operator was more important, so I decided to group them together.

Reply:
Got it.